### ISSUE TITLE
Integrate other 3rd tools, say, nbstripout, with ipypublish.

### ISSUE THREAD

**Issue by hongyi-zhao** [chrisjsewell/ipypublish#138]
Hi,

I really appreciate the wonderful idea of ipypublish. I want to integrate other 3rd tools,  say, [nbstripout](https://github.com/kynan/nbstripout), with ipypublish, any hints for this problem?

Best regards
HY

**Comment by chrisjsewell** [comment]
Hey @hongyi-zhao IIRC you could achieve this with configuration https://ipypublish.readthedocs.io/en/latest/custom_export_config.html#the-configuration-file-format

FYI I'm not doing too much with ipypublish any more, as I'm now working on [myst-nb](https://myst-nb.readthedocs.io) and [jupyter-book](https://github.com/executablebooks/jupyter-book).
So you might want to check them out and let me know if theres any functionality you use in ipypublish that is still lacking in those 😄

**Comment by hongyi-zhao** [comment]
> FYI I'm not doing too much with ipypublish any more, as I'm now working on [myst-nb](https://myst-nb.readthedocs.io) and [jupyter-book](https://github.com/executablebooks/jupyter-book).

Do you mean these projects will substitue the ipypublish in the long run?

**Comment by chrisjsewell** [comment]
> Do you mean these projects will substitue the ipypublish in the long run?

Indeed, with these projects I have basically taken what I learnt writing ipypublish and had the opportunity with [EBP](https://executablebooks.org) to build a new solution from the ground up 😄 

All of the ipypublish features are not yet available in these projects, but the goal is to eventually make that the case
